code/modules/projectiles/ammunition/_firing.dm

```diff
@@ -62,7 +66,7 @@
 /obj/item/ammo_casing/proc/throw_proj(atom/target, turf/targloc, mob/living/user, params, spread, atom/fired_from)
 	var/turf/curloc = get_turf(fired_from)
 	if (!istype(targloc) || !istype(curloc) || !loaded_projectile)
-		return FALSE
+		return
```

Comment:
```suggestion
		return null
```

just so that the intent that this proc will return a reference to something is much more clear